homeassistant/components/binary_sensor/doorbird.py

```diff
@@ -0,0 +1,83 @@
+"""Support for reading binary states from a DoorBird video doorbell."""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import BinarySensorDevice,\
+    PLATFORM_SCHEMA
+from homeassistant.components.doorbird import DOMAIN
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_CONDITIONS, STATE_UNKNOWN
+from homeassistant.util import Throttle
+
+DEPENDENCIES = ['doorbird']
+
+_LOGGER = logging.getLogger(__name__)
+_MIN_UPDATE_INTERVAL = timedelta(milliseconds=250)
+
+SENSOR_TYPES = {
+    "doorbell": {
+        "name": "Doorbell Ringing",
+        "icon": {
+            True: "bell-ring",
+            False: "bell",
+            STATE_UNKNOWN: "bell-outline"
+        }
+    }
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS, default=[]):
```

Comment:
If only `doorbell` is available then this should be the default and the configuration by the user omitted. According to the DoorBird docs there is an additional connector available. That one could be the second type.